pkg/datapath/linux/devices_controller_test.go

```diff
@@ -84,15 +85,37 @@ func TestDevicesController(t *testing.T) {
 
 	logging.SetLogLevelToDebug()
 
-	routeExists := func(routes []*tables.Route, linkIndex int, dst, src string) bool {
+	addrToString := func(addr netip.Addr) string {
+		if !addr.IsValid() {
+			return ""
+		}
+		return addr.String()
+	}
+
+	routeExists := func(routes []*tables.Route, linkIndex int, dst, src, gw string) bool {
 		for _, r := range routes {
-			if r.LinkIndex == linkIndex && r.Dst.String() == dst && r.Src.String() == src {
+			// undefined IP will stringify as "invalid IP", turn them into "".
+			actualDst, actualSrc, actualGw := r.Dst.String(), addrToString(r.Src), addrToString(r.Gw)
+			/*fmt.Printf("%d %s %s %s -- %d %s %s %s\n",
```

Comment:
Stale comment?

Reply:
Sorry about that one, will fix in a followup PR for the same of time